docs/Implementing-easyblocks.rst

```diff
@@ -254,6 +254,39 @@ redefining it entirely, since we call out to the original ``configure_step`` met
 Specific aspects of easyblocks
 ------------------------------
 
+.. _implementing_easyblocks_default_parameters:
+
+Default easyconfig parameters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+All of the parameters which are "set" in an easyconfig file (see :ref:`vsd_avail_easyconfig_params`)
+become key-value pairs in the ``self.cfg`` dictionary.  For instance, where the easyconfig file might
+specify
+
+.. code:: python
+
+  name = 'example'
+  version = '2.5.3'
+  versionsuffix = '-Python-3.7.4'
+
+these are accessible within an easyblock via
+
+.. code:: python
+
+  longform = ''.join(self.cfg['name'],'/',self.cfg['version'],self.cfg['versionsuffix'])
+
+For these "universal" arguments, this mapping is done for you in the ``EasyBlock`` base class, so
```

Comment:
`For these "universal" arguments, an additional mapping is done for you in the ``EasyBlock`` base class, so`